src/control/common/env_utils.go

```diff
@@ -1,5 +1,6 @@
 //
 // (C) Copyright 2019-2024 Intel Corporation.
+// (C) Copyright 2025 Hewlett Packard Enterprise Development LP
```

Comment:
As we update these, we should be removing the (C) as it has no legal meaning in Copyright messages.
Copyright messages must use the word Copyright **or** the special symbol of an R with a circle around it as a shorthand,.
Copyright messages should **not** have both the word Copyright and the special symbol.
The (C) can not be used as a substitute for the special symbol or the word Copyright.
Note copyright messages are not required to enforce a the right of copyright, they just make it easier to do so.

Reply:
FWIW these copyright messages get updated automatically. So if we really want to remove the `(C)` I think it would be better to update the githook to do that instead of doing it manually